Index: extern/remap/src/earcut.hpp
===================================================================
--- extern/remap/src/earcut.hpp	(revision 2694)
+++ extern/remap/src/earcut.hpp	(working copy)
@@ -7,7 +7,7 @@
 #include <vector>
 #include <limits>
 //#include <tuple>
-//#include <cstdint> 
+#include <cstdint> 
 //#include <cstddef>
 
 namespace mapbox {
Index: extern/remap/src/elt.hpp
===================================================================
--- extern/remap/src/elt.hpp	(revision 2694)
+++ extern/remap/src/elt.hpp	(working copy)
@@ -3,6 +3,7 @@
 #include <list>
 #include "triple.hpp"
 #include <vector>
+#include <array>
 
 #define NMAX 0 /**< maximum number of vertices for polygons */
 
Index: src/xios_spl.hpp
===================================================================
--- src/xios_spl.hpp	(revision 2694)
+++ src/xios_spl.hpp	(working copy)
@@ -16,6 +16,7 @@
 
 // Conteneurs.
 #include <vector>
+#include <array>
 #include <set>
 #include <stack>
 #include <list>
